client/src/containers/map/popup/item.tsx

```diff
@@ -75,41 +124,54 @@ const PopupItem = ({ id }: PopupItemProps) => {
       map?.off('render', handleMapRender);
     };
   }, [map, handleMapRender]);
-
+  const noData = <div className="text-xs">has No data</div>;
   return (
-    <div className="p-4">
-      <div className="space-y-3">
-        <h2 className="text-base font-semibold">{attributes.title}</h2>
-
-        <ContentLoader
-          data={data?.data}
-          isFetching={isFetching || (!rendered && !DATA_REF.current)}
-          isFetched={isFetched && (rendered || !!DATA_REF.current)}
-          isError={isError}
-          isPlaceholderData={isPlaceholderData}
-          skeletonClassName="h-20 w-[250px]"
-        >
-          <dl className="space-y-2">
-            {click &&
-              !!DATA &&
-              click.values.map((v) => {
-                return (
-                  <div key={v.key}>
-                    <dt className="text-xs font-semibold uppercase">{v.label || v.key}:</dt>
-                    <dd className="text-sm">
-                      {format({
-                        id: v.format?.id,
-                        value: DATA[v.key],
-                        options: v.format?.options,
-                      })}
-                    </dd>
-                  </div>
-                );
-              })}
-            {click && !DATA && <div className="text-xs">No data</div>}
-          </dl>
-        </ContentLoader>
-      </div>
+    <div className="space-y-3 pt-2 text-gray-700 first:pt-0">
```

Comment:
I think the design has changed since you implemented this. Can you review it? https://www.figma.com/file/DBXFTLd6nFBmg6fFE6P2cX/ORCaSa-Design?type=design&node-id=5744-20206&mode=design&t=3RFFcbTkLQPAJEhZ-4